sandbox.sh

```diff
@@ -94,11 +94,12 @@ EOF
   echo "]"
 }
 
-# Business!
+# Business happen here!
```

Comment:
These comments are not very useful.

Reply:
Oh but I understand now that most of your comments are actually questions/personal observations, and that you're not asking for them to actually get merged. Sorry!